### ISSUE TITLE
cooperator module demo data : rely on standard module demo data

### ISSUE THREAD

**Issue by robinkeunen** [coopiteasy/vertical-cooperative#279]
cf title. For example : do not create new partners, use marc_demo.

Also remove inconsistent demo. eg. cooperators w/o cooperator number nor share lines and stuff

**Comment by robinkeunen** [comment]
cf #290

**Comment by victor-champonnois** [comment]
@robinkeunen do we move this issue to OCA ? Is it still relevant now that we know more about test best practices ?

**Comment by robinkeunen** [comment]
Yes I think it is still relevant. There are plenty of demo data that could be based on standard data. It is different than creating test data in the unit tests instead of using demo data.
